src/components/dashboard/main-content/course-enrollments/course-cards/SavedForLaterCourseCard.jsx

```diff
@@ -63,12 +65,9 @@ const SavedForLaterCourseCard = (props) => {
       courseRunId: response.courseRunId,
       newStatus: response.courseRunStatus,
     });
+    addCourseEnrollmentStatusChangeAlert(COURSE_STATUSES.inProgress);
     navigate('.', {
       replace: true,
-      state: {
-        markedSavedForLaterSuccess: false,
-        markedInProgressSuccess: true,
-      },
     });
```

Comment:
[sanity check] Is this `navigate` call needed at all if no longer updating the location `state`? It's largely a no-op at this point, I believe.

Reply:
The navigate in this case pushes the user to the top of the page indicating the alert. For the most part this is necessary if the user happens to have a lot of enrollments and the user is not immediately aware of the alert because they have scrolled past the point where the alert will render.

Curious for your feedback for a user centric approach on what makes the most sense: allow the navigate (or use a scroll to top instead) or remove altogether?

Reply:
Decision from related Slack thread: we'll continue to scroll to the top upon success, but with `scrollTo` vs. `navigate`.

In the future, we may start a conversation with UX to switch the success alert to a `Toast` and revisit the scroll behavior.